src/benchmarks/micro/corefx/System.Text.Json/Utf8JsonReader/Utf8JsonReader.SingleSegment.Tests.cs

```diff
@@ -0,0 +1,132 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using BenchmarkDotNet.Attributes;
+using MicroBenchmarks;
+using MicroBenchmarks.Serializers;
+using System.Collections.Generic;
+using System.IO;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace System.Text.Json.Serialization.Tests
+{
+    [BenchmarkCategory(Categories.CoreFX, Categories.JSON)]
+    public partial class Utf8JsonReaderTests
```

Comment:
I would update the class name to indicate this is specifically about comments (similar to the file name change).
Maybe: Utf8JsonReader_Comments

```suggestion
    public partial class Utf8JsonReader_Comments
```